.github/workflows/tests.yml

```diff
@@ -35,8 +31,67 @@ jobs:
       - name: "Run C/C++ linter"
         run: ./bin/run_clang_tidy.sh
 
+  cpp-funcs:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-20.04
+    container:
+      image: faasm/cpp-sysroot:0.0.22
```

Comment:
Annoyingly we have another two versions to maintain in here now, but we can worry about that later